subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -517,7 +518,25 @@ int nrf_cloud_rest_location_get(struct nrf_cloud_rest_context *const rest_ctx,
 	memset(&resp, 0, sizeof(resp));
 	init_rest_client_request(rest_ctx, &req, HTTP_POST);
 
-	req.url = request->disable_response ? API_GET_LOCATION_NO_REPLY : API_GET_LOCATION;
+	char url[sizeof(API_GET_LOCATION_CONFIG) + sizeof("false") * 3];
+	bool do_reply;
+
+	if (request->config) {
+		ret = snprintk(url, sizeof(url) - 1, API_GET_LOCATION_CONFIG,
+			       request->config->do_reply ? "true" : "false",
+			       request->config->fallback ? "true" : "false",
+			       request->config->hi_conf  ? "true" : "false");
```

Comment:
If we wanted to optimize the size of the REST URL, you could add just those params that are non-default. I'm not sure you want to go there though...

Reply:
Yes, I drew the line at that point.